src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -1323,6 +1323,11 @@ public TabularData getFailureDetectorPhilValues()
             throw new RuntimeException(e);
         }
     }
+
+    public MessagingServiceMBean getMsProxy()
```

Comment:
can we use a better name, something like `getMessagingService` or `getMessagingServiceProxy`?

Reply:
The method name, `getMsProxy`, is not good. The reason to do so is to follow the existing method naming convention in the class.

Reply:
there are other naming conversion as well... I guess this class isn't too consistent =)